modules/battery_manager/help/battery_manager.md

```diff
@@ -0,0 +1,54 @@
+# Battery Manager
+*For Rida: Add what each value in the test_battery does and what happens if it's left empty*
+The Battery Manager module serves as a front-end for manipulating the Test Battery.
+This includes browsing, adding, editing, activating, and deactivating entries.
+
+## Searching for an entry in the Test Battery
+Under the `Browse` tab, use the `Selection Filters` to search for entries by fields such as:
+`Instrument`,
+`Minimum age (days)`,
+`Maximum age (days)`,
+`Stage`,
+`Subproject`,
+`Visit Label`,
+`Site`,
+`First Visit`,
+`Instrument Order`,
+`Active`.
```

Comment:
The help should probably describe the rules for how these interact somewhere.

Reply:
I'm not sure what you mean

Reply:
@ridz1208

Reply:
@HenriRabalais copy the stuff from the readme